frontend/src/app/testQueue/addToQueue/AddToQueueSearch.jsx

```diff
@@ -113,11 +114,25 @@ const AddToQueueSearchBox = ({ refetchQueue, facilityId, patientsInQueue }) => {
     fetchPolicy: "no-cache",
     variables: { facilityId, namePrefixMatch: queryString },
   });
+
   const [mutationError, updateMutationError] = useState(null);
+  const [showSuggestion, setShowSuggestion] = useState(true);
+
   const [addPatientToQueue] = useMutation(ADD_PATIENT_TO_QUEUE);
   const [updateAoe] = useMutation(UPDATE_AOE);
 
   const allowQuery = debounced.length >= MIN_SEARCH_CHARACTER_COUNT;
+  const showDropdown = useMemo(() => allowQuery && showSuggestion, [
+    allowQuery,
+    showSuggestion,
+  ]);
+
+  const dropDownRef = useRef(null);
+  const hideOnOutsideClick = () => {
+    setShowSuggestion(false);
+  };
```

Comment:
Recommend wrapping this in a `useCallback` so the `useOutsideClick` hook can have a full dependency array

```javascript
const hideOnOutsideClick = useCallback(() => {
  setShowSuggestion(false);
}, []);
```

Reply:
These and previous remarks - I totally agree with that. I'll fix it in a few minutes.